src/agent/rustjail/src/cgroups/fs/mod.rs

```diff
@@ -1404,9 +1406,9 @@ impl Manager {
             }
 
             let p = if value == "/" {
-                format!("{}{}", mnt.unwrap(), cpath)
+                format!("{}{}", mnt.unwrap(), cgroup_path)
```

Comment:
Could you also change these `unwrap()`'s to `map_err()?`'s?

Reply:
Hi @jodh-intel 

Those unwraps are on Option variables and they had been tested with is_some(), so here it's safe to unwrap on them.

Reply:
what happen if `cpath` doesn't start with '/' ?, `cgroup_path` will be empty, so `p` will be equal to `mnt.unwrap()`, is this expected ?

Reply:
Hi @devimc  good point, I had updated.

Reply:
What about just using `format!("{}/{}", xxx, xxx)`, adding an extra `/` in between in all cases?

Reply:
Yeah, that's an option, which let the kernel do the '/' deduplication.